app/views/passwords/new.html.erb

```diff
@@ -1,19 +1,19 @@
 <!--Descriptive page name, messages and instructions-->
 <h3 class="heading"><%= ts("Forgotten your password?") %></h3>
 <p>
-  <%= ts("If you've forgotten your password, we can send you a temporary one.") %> 
+  <%= ts("If you've forgotten your password, we can send you a temporary one.") %>
   <%= ts("Please tell us the user name or email address you used when you signed up for your Archive account.") %>
 </p>
 <p>
-  <%= ts("If the temporary password doesn't work, please visit <a href=\"http://archiveofourown.org/admin_posts/1277\">Problems When Logging In</a> for help.".html_safe) %>
+  <%= ts("If the temporary password doesn't work, please visit %{problems_when_logging_in} for help.", :problems_when_logging_in => link_to(ts("Problems When Logging In"), admin_posts_path("1277"))) %>
```

Comment:
will do! as soon as i have a minute i'll get to work on this. i'm moving in a week but after that i should have some time.